screens/change_pokemon/change_pokemon.lua

```diff
@@ -35,7 +35,7 @@ M.config = {
 		[1]=hash("change_pokemon/nature"), [2]=hash("change_pokemon/extra") ,
 		[3]=hash("change_pokemon/asi/root"), [4]=hash("change_pokemon/moves"),
 		[5]=hash("change_pokemon/abilities"), [6]=hash("change_pokemon/feats"),
-		[7]=hash("change_pokemon/held_item")
+		[7]=hash("change_pokemon/held_item"), [8]=hash("change_pokemon/custom_asi/root")
 	},
 	start = vmath.vector3(0, -110, 0),
 	[hash("change_pokemon/asi/root")] = {open=vmath.vector3(720, 420, 0), closed=vmath.vector3(720, 85, 0), active=true},
```

Comment:
This system is really quite cool btw. But do be aware that I added a new section (higher up) for variants, so you may notice a conflict when merging from master.